src/api/routers/participantsRouter.ts

```diff
@@ -161,34 +166,40 @@ export function createParticipantsRouter() {
     }
   });
 
+  participantsRouter.use('/:participantId', enrichCurrentUser);
+
   participantsRouter.put(
     '/:participantId/approve',
     isApproverCheck,
-    async (req: ParticipantRequest, res: Response) => {
-      const { participant } = req;
+    async (req: UserParticipantRequest, res: Response) => {
+      const { participant, user } = req;
       const traceId = getTraceId(req);
       const data = {
         ...ParticipantApprovalPartial.parse(req.body),
         status: ParticipantStatus.Approved,
       };
 
-      const auditTrail = await insertApproveAccountAuditTrail(
-        participant!,
-        req.auth?.payload?.email as string,
-        data
-      );
+      const auditTrail = await insertApproveAccountAuditTrail(participant!, user!, data);
       const kcAdminClient = await getKcAdminClient();
       const users = await getAllUserFromParticipant(participant!);
+      // if there are no users, send email to the approver
+      const emailRecipient = users.length > 0 ? users : [user!];
       await setSiteClientTypes(data);
       await Promise.all(
-        users.map((user) =>
-          assignClientRoleToUser(kcAdminClient, user.email, 'api-participant-member')
+        users.map((currentUser) =>
+          assignClientRoleToUser(kcAdminClient, currentUser.email, 'api-participant-member')
         )
       );
+
       await updateParticipantAndTypesAndRoles(participant!, data);
-      await sendParticipantApprovedEmail(users, traceId);
+      await sendParticipantApprovedEmail(emailRecipient, traceId);
       await updateAuditTrailToProceed(auditTrail.id);
-      return res.sendStatus(200);
+
+      const approvalResponse: ParticipantApprovalResponse = {
```

Comment:
new object type that just has users for now but felt like the better thing to return and can have other props added later